packages/playwright-core/src/containers/index.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-console */
+
+import path from 'path';
+import { spawnAsync } from '../utils/spawnAsync';
+import { createGuid } from '../utils';
+import type { Command } from '../utilsBundle';
+import { debug } from '../utilsBundle';
+import type { AddressInfo } from 'net';
+import http from 'http';
+import { selfDestruct } from '../cli/driver';
+import { PlaywrightServer } from '../remote/playwrightServer';
+
+const { ProxyServer } = require('../third_party/http_proxy.js');
+const debugLog = debug('pw:proxy');
+
+export function addContainerCLI(program: Command) {
+  const ctrCommand = program.command('container', { hidden: true })
+      .description(`Manage container integration (EXPERIMENTAL)`);
+
+  ctrCommand.command('install-services', { hidden: true })
+      .description('install services required to run container agent')
+      .action(async function() {
+        const { code } = await spawnAsync('bash', [path.join(__dirname, 'container_install_deps.sh')], { stdio: 'inherit' });
```

Comment:
`shell: true` ?

Reply:
this already runs bash, so `shell: true` should not be necessary